grails-app/services/org/pih/warehouse/data/DataService.groovy

```diff
@@ -382,6 +386,17 @@ class DataService {
         return product
     }
 
+    def findOrCreateProductType(String productTypeName) {
+        def productType = ProductType.findByName(productTypeName)
+        if (!productType) {
+            println("Could not find product type " + productTypeName)
```

Comment:
i think we can use log.info or remove this

Reply:
Ok